lang/pypy/Portfile

```diff
@@ -66,12 +65,18 @@ long_description \
     instead of CPython is speed, as it runs generally faster.
 
 post-patch {
-    reinplace "s+('/sw/', '/opt/local/')+('__PREFIX__',)+g" \
+    # the PyPy sources search for  dependancies in /opt/local and /sw,
+    # yet we only want to search  ${prefix} -- the hackery below fixes
+    # that, but without triggering warnings about /opt/local from 'port lint'
+    set default_prefix "/opt"
+    append default_prefix "/local"
+
+    reinplace "s|('/sw/', '${default_prefix}/')|('__PREFIX__',)|" \
         ${worksrcpath}/pypy/tool/cpyext/extbuild.py
 
     # sanity check, useful when upgrading
-    if { ![catch {exec grep -lwre "/sw" -e "/opt/local" ${worksrcpath}}] } {
-        return -code error "didn't catch all references to /sw or /opt/local!"
+    if { ![catch {exec grep -lwre "/sw" -e ${default_prefix} ${worksrcpath}}] } {
+        return -code error "didn't catch all references to /sw or ${default_prefix}!"
```

Comment:
Is this block really necessary? Have the developers really been making so frequent and sweeping changes to their code that this has actually been an issue? We have tons of other ports that use the patch-with-placeholder-then-reinplace strategy that haven't felt the need to add such sanity checks.